Cakes-Co/style.css

```diff
@@ -1 +1,245 @@
-/*  Add your styling here */
+* {
+    margin: 0;
+    padding: 0;
+  }
+  body {
+    font-family: "Roboto", sans-serif;
+    font-size: 4vw;
+    background-image: linear-gradient(rgb(243, 207, 232), rgb(247, 243, 168), rgb(255, 126, 203));
+     
+  }
+  h1 {
+    font-size: 2.4em;
+    font-weight: 500;
+  }
+  h4 {
+    font-size: 1.2em;
+    font-weight: 400;
+    font-style: italic;
```

Comment:
I like what you're doing here,setting the base styles for diffrent sorts of elements